packages/venia-ui/lib/fragments/productDetails.graphql

```diff
@@ -21,7 +21,7 @@ fragment ProductDetails on ProductInterface {
     }
     meta_title @include(if: $onServer)
     meta_keyword @include(if: $onServer)
-    meta_description @include(if: $onServer)
```

Comment:
We now want `meta_description` regardless of whether we are doing SSR. I assume there is still a path to use this in SSR as there are templates in our repo that use these meta fields but I couldn't figure out how exactly.